src/plugins/data/public/search/session/sessions_client.ts

```diff
@@ -55,7 +54,7 @@ export class SessionsClient {
     restoreState: Record<string, unknown>;
     urlGeneratorId: string;
     sessionId: string;
-  }): Promise<SavedObject<BackgroundSessionSavedObjectAttributes>> {
+  }): Promise<SavedObject> {
```

Comment:
Is the intention to use this client in the sessions management UI? 

I can do that, but it looks like the client methods are less type-aware after this PR